app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -730,29 +712,21 @@ public function favorite(Request $request, Organization $suborganization, Projec
     }
 
     /**
-     * Get All Favorite Projects
-     *
-     * @urlParam suborganization required The Id of a suborganization Example: 1
```

Comment:
Need to add docs as per following conventions
https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api#specifying-request-parameters

Need "@urlParam suborganization"